packaging/greenboot/green.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -ex
+
+HEALTH=healthy
+
```

Comment:
Need to check that the script is running from the `root` account. See how it's done in the other greenboot scripts and let's use the same code / messages.